MerlinAU.sh

```diff
@@ -4218,25 +4227,30 @@ Please manually update to version $minimum_supported_version or higher to use th
     fi
 
     ##------------------------------------------##
-    ## Modified by ExtremeFiretop [2024-Feb-03] ##
+    ## Modified by ExtremeFiretop [2024-May-21] ##
```

Comment:
Changed the SHA256 logic to download from the Merlin website instead of comparing to the zip contents.
As requested by @dave14305

Reply:
> Changed the SHA256 logic to download from the Merlin website instead of comparing to the zip contents. As requested by @dave14305
>
Excellent!!